Jetsurvey/app/src/main/java/com/example/compose/jetsurvey/survey/SurveyQuestions.kt

```diff
@@ -235,22 +233,18 @@ private fun QuestionContent(
 
 @Composable
 private fun QuestionTitle(@StringRes title: Int) {
-    val backgroundColor = if (MaterialTheme.colors.isLight) {
-        MaterialTheme.colors.onSurface.copy(alpha = 0.04f)
-    } else {
-        MaterialTheme.colors.onSurface.copy(alpha = 0.06f)
-    }
     Row(
         modifier = Modifier
             .fillMaxWidth()
             .background(
-                color = backgroundColor,
+                color = MaterialTheme.colorScheme.inverseOnSurface,
                 shape = MaterialTheme.shapes.small
             )
     ) {
         Text(
             text = stringResource(id = title),
-            style = MaterialTheme.typography.subtitle1,
+            style = MaterialTheme.typography.titleMedium,
+            color = MaterialTheme.colorScheme.onSurface.copy(alpha = 0.87f),
```

Comment:
I'm still a bit unsure about these alpha values, but I see they're coming from the designs. In general it would be great to have them as constants somewhere.

Reply:
Yeah, I wasn't a fan of the alphas but at least having the constants makes this cleaner.